data/common.js

```diff
@@ -0,0 +1,4 @@
+// ATTENTION: Keep in sync with C++ function of the same name in filesystem.cpp and `Generator::escapeAttrForFilename`
+function replace_invalid_filename_chars(str) {
+    return str.replace(new RegExp(':', 'g'), '_');
+}
```

Comment:
would be nice if older generated file would still work with the newer javascript.
older generated files do not include this common.js.  So maybe we need a fallback for when the function does not exist.

Note: I've been trying to keep only one .js included. this is why symbol.html has some duplicated code.
I've been wanting to use a build system to concatenate and minify files, but never got around to do that.

Reply:
Yes, a global var would work. Maybe with a version number or so.

Reply:
Ah yes, because that is not generated but shared in the data folder. :shrug: